en/docs/services/create-your-first-service.md

```diff
@@ -18,70 +18,75 @@ To create the Choreo service, follow the procedure below:
 
 4. In the **Configure API Trigger** form, select **GET** as the HTTP method, and enter `activecases` as the relative path from host.
 
-    ![Resource Configuration](../assets/img/services/configure-api-trigger.png)
+    ![Resource Configuration](/en/assets/img/services/configure-api-trigger.png)
```

Comment:
Use relative paths. Check all the places. Changes were added in https://github.com/wso2/docs-choreo-dev/pull/63/files